descriptors/ModuleDescriptor-template.json

```diff
@@ -200,6 +200,37 @@
         }
       ]
     },
+    {
+      "id": "orders-storage.order-invoice-relationship",
```

Comment:
I think, interface naming "orders-storage.order-invoice-relationship**s**" should be used. Also permissions naming should be changed to use "order-invoice-relationship**s**"

Reply:
Done. Check again please.